src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqFilterSpark.java

```diff
@@ -99,6 +109,7 @@ protected void runTool(final JavaSparkContext ctx) {
         } else {
             logger.info("No unpaired reads to write - BAM will not be written.");
         }
+
```

Comment:
Remove blank line (if need to change this file for something else, otherwise forget it).

Reply:
No changes to this file